.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+      max-parallel: 1
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -U setuptools
+        pip install -r requirements.txt
+        pip install -e .
+        pip install codecov
+    - name: Run memcached service
+      uses: jkeys089/actions-memcached@master
+    - name: Run tests
+      run: |
+        make cov
+        python setup.py check -rm
+    - name: Upload coverage
+      uses: codecov/codecov-action@v1
+      with:
+        file: ./coverage.xml
+        flags: unit
+
+  deploy:
+    name: Deploy
+    environment: release
```

Comment:
@webknjaz Can you setup the release environment please?

Reply:
@asvetlov I've found another PyPI package where I'm not added as an owner.

Reply:
@Dreamsorcerer done